homeassistant/components/google/__init__.py

```diff
@@ -69,6 +70,7 @@
     vol.Optional(CONF_OFFSET): cv.string,
     vol.Optional(CONF_SEARCH): cv.string,
     vol.Optional(CONF_TRACK): cv.boolean,
+    vol.Optional(CONF_MAX_RESULTS): cv.string,
```

Comment:
Why would this be a string?

Reply:
I believe there's a cv.int you could use.

Reply:
Found a positive_int. Fixed